### DISCUSSION TITLE
Unable to use remote ID connection under the same Wi-Fi

### DISCUSSION THREAD

**Discussion by Anber55** [rustdesk/rustdesk#9655] [General]
Why can't the iPhone connect to the Mac using the remote ID under the same Wi-Fi, it has to connect to the Mac using the IP address to be successful
If you are not on the same Wi-Fi, iPhone can't connect to Mac with IP address, you have to connect to Mac with Remote ID to succeed.
Is it possible to connect to the Mac via Remote ID regardless of whether it's on the same Wi-Fi or not, or what would need to be changed if I wanted to do this, appreciated!

**Comment by xlionjuan** [comment]
[If an app would like to connect to devices on your local network](https://support.apple.com/en-us/102229)
